client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -162,45 +194,44 @@ class EditContactInfoFormCard extends React.Component {
 				},
 				{
 					action: 'confirm',
-					label: this.props.translate( 'Confirm' ),
+					label: this.props.translate( 'Request Confirmation' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **45** times:
`translate( 'Confirmation' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).